src/applications/mhv-secure-messaging/components/MessageThread/MessageThreadItem.jsx

```diff
@@ -111,7 +115,14 @@ const MessageThreadItem = props => {
       }`}
     >
       <h3 slot="headline">
-        {isDraft ? 'DRAFT' : dateFormat(sentDate, 'MMMM D [at] h:mm a z')}
+        {isDraft
+          ? 'DRAFT'
+          : dateFormat(
+              sentDate,
+              readReceiptsEnabled
+                ? 'MMMM D, YYYY [at] h:mm a z'
+                : 'MMMM D [at] h:mm a z',
+            )}

```

Comment:
this change probably does not need to be feature flagged. Seems like just a defect fix for me, regardless of readReceipts work